### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by AagmanSuri**

### CODE REVIEW

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  width: 100px;
+  object-fit: cover;
+}
+
+.online {
+  background-color: green;
+  height: 1.3rem;
+  width: 1.3rem;
+  border-radius: 50%;
+  border: 0.2rem solid white;
```
Comment (aiman-ansari) [inline Components/Badge/badge.css]: Use color variable here.

File: Components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  font-size: 1.6rem;
+}
+
+.sub-title {
+  font-size: 2.72rem;
+  font-weight: 800;
+}
+
+hr {
+  background-color: lightgray;
```
Comment (aiman-ansari) [inline Components/Badge/badge.css]: Use color variable.

File: Components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  height: 90px;
+  width: 90px;
+}
+
+.lg {
+  height: 120px;
+  width: 120px;
+}
+
+.square {
```
Comment (aiman-ansari) [inline Components/Avatar/avatar.css:33]: Instead of square you can name it avatar-square

**Review by aiman-ansari — CHANGES_REQUESTED** [review]
In Avatar.css and badge.css till the margin-top-1 you have written same code which you have already styled In component.css Remove same code and link component.css in both files
Also there are some places where you have used px so convert it into rem

**Comment by PuneetP16** [comment]
image
The code in the carbon is not structured in a proper format.

File: Components/Avatar/avatar.css [hunk 4]
```diff
@@ -55,9 +57,41 @@ hr {
 .margin-top-1 {
   margin-top: 33rem;
 }
-.avatar {
+
+.avatar-container {
   display: flex;
-  background-color: lightskyblue;
+  min-height: 20rem;
```
Comment (PuneetP16) [inline Components/Avatar/avatar.css]: use color variable.

File: Components/Avatar/avatar.html [hunk 5]
```diff
@@ -93,11 +93,50 @@
...
                 <hr class="margin-bottom-1">
 
+                <!-- Square Avatar --><span class="sub-title">Square Avatar</span>
+                <p class="p-content">Adding a class 'square' to existing class, will generate a square shaped avatar.
+                </p>
+
+                <div class="avatar-container">
+                    <img class="avatar sm square" src="https://bit.ly/3J94zSO">
+                    <img class="avatar md square" src="https://bit.ly/3ou569I">
+                    <img class="avatar lg square" src="https://bit.ly/34xfbf4">
```
Comment (PuneetP16) [inline Components/Avatar/avatar.html]: add size classes to the end.

**Review by PuneetP16 — APPROVED** [review]
Good to merge after necessary changes

File: Components/Badge/badge.css (hunk 1 above)
Comment (AagmanSuri) [reply Components/Badge/badge.css]: I have changed them to var and removed the common css . Thanks for reviewing

File: Components/Avatar/avatar.css (hunk 4 above)
Comment (AagmanSuri) [reply Components/Avatar/avatar.css]: I have changed them to var and structured the code correctly. Thanks for reviewing

File: Components/Avatar/avatar.html (hunk 5 above)
Comment (AagmanSuri) [reply Components/Avatar/avatar.html]: I have added the sizes to the end . Thanks for reviewing

**Pull request merged**